docs/USER_GROUP_API.md

```diff
@@ -214,10 +216,11 @@ error EnumerationNotSupported()
 The `GetUserRecord` method looks up or enumerates a user record.
 If the `uid` parameter is set it specifies the numeric UNIX UID to search for.
 If the `userName` parameter is set it specifies the name of the user to search for.
-Typically, only one of the two parameters are set, depending whether a
-look-up by UID or by name is desired.
-However, clients may also specify both parameters, in which case a record matching both will be returned, and if only
-one exists that matches one of the two parameters but not the other an error of `ConflictingRecordFound` is returned.
+If the `uuid` parameter is set it specifies the UUID of the user to search for.
+Typically, only one of the three parameters are set, depending whether a
+look-up by UID, name or UUID is desired.
```

Comment:
For enumerations with at least three elements, please add a comma before "or" or "and" (So-called "oxford comma".)